src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Ball {
+    public static void ball(String color, int number) {
+
+    }
```

Comment:
Remove this
You ignore incoming arguments
Create a constructor for color and number, add those as two fields of Ball class